corehq/apps/app_manager/tests/test_suite_registry_search.py

```diff
@@ -46,7 +46,7 @@ class RemoteRequestSuiteTest(SimpleTestCase, TestXmlMixin, SuiteMixin):
     def setUp(self):
         self.app = Application.new_app(DOMAIN, "Untitled Application")
         self.app._id = '123'
-        self.app.build_spec = BuildSpec(version='2.53.0', build_number=1)
+        self.app.build_spec = BuildSpec(version='2.52.0', build_number=1)
```

Comment:
I thought registries/case search were 2.53-specific, but according to @orangejenny 's spec it seems like that's not the case?

Reply:
Case search isn't, but registries are, yes, so this should indeed be 2.53. I'm surprised these tests are passing.